src/main/java/de/unistuttgart/informatik/fius/icge/simulation/Simulation.java

```diff
@@ -435,4 +434,20 @@ private synchronized void tick(Runnable afterwards) {
             super(sim);
         }
     }
+
+    /**
+     * Add an enity to the list
+     * @param entity
+     */
+    public void addEntity (Entity entity) {
+        this._entities.add(entity);
+    }

```

Comment:
I already wrote a comment about why we need to have an `Entity`-list. Without that list, this method wouldn't be needed.